queries.sql

```diff
@@ -8,3 +8,84 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/*The BEGIN statement starts a new transaction. The UPDATE statement updates all rows in the 'animals' table
+by setting the 'species' column to 'unspecified'. The SELECT statement verifies that the change was made
+by selecting all rows where the 'species' column is 'unspecified'. The ROLLBACK statement rolls back the transaction,
+undoing the changes made by the UPDATE statement. The final SELECT statement verifies that the 'species' column
+went back to its original state by selecting all rows where the 'species' column is not 'unspecified'.*/
```

Comment:
- [ ] [**OPTIONAL**] Good job applying the changes requested by the previous Code reviewer (CR). Also it is something good to write comments , however It could be better to keep it clear and avoid explaning the code 😄 , Kindly take care of it next time 👍 . I will suggest you to check [this link](https://swimm.io/learn/code-collaboration/comments-in-code-best-practices-and-mistakes-to-avoid#:~:text=Use%20clear%20and%20concise%20language,notes%2C%20remarks%2C%20or%20jokes.) for further details about  **comments good practices**